### DISCUSSION TITLE
Website Plugins Link Broken

### DISCUSSION THREAD

**Discussion by setchy** [ohmyzsh/ohmyzsh#11717] [Help]
The `PLUGINS` nav link in the header of https://ohmyz.sh/ is pointing to https://github.com/ohmyzsh/ohmyzsh/wiki/Themes instead of https://github.com/ohmyzsh/ohmyzsh/wiki/Plugins

![Screenshot 2023-05-30 at 8 45 43 AM](https://github.com/ohmyzsh/ohmyzsh/assets/386277/0910310b-393c-4f9f-96ac-fee6430912d2)

**Comment by carlosala** [comment]
Already tracked in #11716
